person.rb

```diff
@@ -9,10 +9,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
If we set `rentals` as an `attr_reader`, it means we can only read rentals but not write rentals, ultimately we need rentals to be read and writable

- [ ] Kindly update this attribute

Reply:
Understood. I'll implement that. Thank you.

Reply:
- Well resolved 👍🏽